tvaultuiv2/src/views/private/service-accounts/OnBoardForm/index.js

```diff
@@ -750,8 +763,10 @@ const OnBoardForm = (props) => {
                   </InfoLine>
                 </HeaderInfoWrapper>
                 <ServiceAcoountHelp
-                  title="view more"
+                  titleMore="view more"
```

Comment:
View More and View Less should be capitalize like https://app.zeplin.io/project/5d51b7c11ced2b350a928a7f/screen/5f65559f716f3322020eba87